src/components/Toolbar.js

```diff
@@ -23,16 +24,28 @@ export default class Toolbar extends Component {
         const { navigator } = this.context;
         const { onIconPress } = this.props;
 
+        this.title = navigator && navigator.currentRoute ? navigator.currentRoute.title : 'Refugee Info'
         return (
-            <MaterialToolbar
-                icon={navigator && navigator.isChild ? 'keyboard-backspace' : 'menu'}
-                onIconPress={() => {navigator && navigator.isChild ? navigator.back() : onIconPress();}}
-                rightIconStyle={{
-                    margin: 10
-                }}
-                testID="test-id-toolbar"
-                title={navigator && navigator.currentRoute ? navigator.currentRoute.title : 'Refugee Info'}
-            />
+            <ToolbarAndroid
```

Comment:
This line seems to like a potential problem with running app on android. I guess it's not possible to use ```MaterialToolbar```, right?